fastlane/metadata/android/ru-RU/changelogs/default.txt

```diff
@@ -1,5 +1,5 @@
-25,2:
+25.2:
```

Comment:
How did this end up with a `,`?

Reply:
I think this is editable by the translators?